app/javascript/components/UI/ShoutoutModal.js

```diff
@@ -0,0 +1,66 @@
+import React, {Fragment,  useState} from 'react'
+import ReactDOM from 'react-dom'
+import RichInputElement from "./rich-text/RichInputElement";
+import {apiRequest} from "../requests/axios_requests";
+import {isEmpty} from "../helpers/helpers";
+
+const ShoutoutModal = ({ onClose, data, setData: setDataInDB, editObj = {} }) => {
+
+  const BackDrop = () => {
+    return <div className='backdrop' onClick={ onClose }/>
+  }
+
+  const ModalOverlay = () =>{
+    const [ richText ] = useState( isEmpty(editObj) ? '' : editObj.rich_text )
+    const [ idEditText ] =  useState( isEmpty(editObj) ? null : editObj.id )
```

Comment:
@VitaliiShevchenko hook `useState`  assumes you are using a value and a function, for example:
`const [richText , setRichText] = useState( isEmpty(editObj) ? '' : editObj.rich_text )`
I think you can change it to:
`const  richText  = isEmpty(editObj) ? '' : editObj.rich_text `
........